src/ApiService/ApiService/Functions/ValidateScriban.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+
+namespace Microsoft.OneFuzz.Service.Functions;
+
+public class ValidateScriban {
+    private readonly ILogTracer _log;
+    private readonly IOnefuzzContext _context;
+    public ValidateScriban(ILogTracer log, IOnefuzzContext context) {
+        _log = log;
+        _context = context;
+    }
+
+    private async Async.Task<HttpResponseData> Post(HttpRequestData req) {
+        var request = await RequestHandling.ParseRequest<TemplateValidationPost>(req);
+        if (!request.IsOk) {
+            return await _context.RequestHandling.NotOk(req, request.ErrorV, "ValidateTemplate");
+        }
+
+        var instanceUrl = _context.ServiceConfiguration.OneFuzzInstance!;
+
+        var (renderer, templateRenderContext) = await GenerateTemplateRenderContext(request.OkV.Context);
+
+        var renderedTemaplate = await renderer.Render(request.OkV.Template, new Uri(instanceUrl), enableJinjaAdapter: false);
+
+        var response = new TemplateValidationResponse(
+            renderedTemaplate,
+            templateRenderContext
+        );
+
+        return await RequestHandling.Ok(req, response);
```

Comment:
What kind of error messages will people see if the rendering fails?

Reply:
In the current state of the PR, the invalid templates simply won't render and there won't be an error message.

For example, given the input:
`This exists: {{ report.target_executable }} but this doesn't: {{ report.NOT_A_REAL_PROPERTY }}`

The result:
`This exists: target.exe but this doesn't: `

From reading the scriban docs, there is a way to enable 'strict' variable resolution which would cause the above example to throw an exception which we could catch and return in the response. I'm investigating this option to see if the exception information is valuable or not.

Reply:
Discussed offline, we'll be validating the template using strict mode before inserting it into the database. This way we don't need to worry about notification creation time because only strictly validated templates should be in the database. This means we'll also use strict mode in the validation endpoint.